wrappers/core18.go

```diff
@@ -204,5 +204,71 @@ func writeSnapdServicesOnCore(s *snap.Info, inter interacter) error {
 		return err
 	}
 
+	// Handle the user services
+	if err := writeSnapdUserServicesOnCore(s, inter); err != nil {
+		return err
+	}
+
```

Comment:
Looking at this: what is the behavior of old snapd reexecuting into new snapd on a classic distribution? Will the new feature set work or will it require a package update?

Reply:
These lines will never be executed on a classic system.  See the `release.OnClassic` check at the start of the function.